pages.zh/common/rsync.md

```diff
@@ -3,34 +3,34 @@
 > 一种快速，通用，远程（和本地）文件复制工具。
 > 更多信息：<https://man7.org/linux/man-pages/man1/rsync.1.html>。
 
-- 从本地传输文件到远程主机：
+- 从本地传输文件到远程主机:
```

Comment:
If we don't have a project-wise rule requiring English colon in bullet point, I believe Chinese colon is still preferred here. Is there any reason you are using different colons in description and bullet point?

Reply:
Yes @gyli, there is a reason.  I would like to use the Chinese colon because I know that that's the best one, and that's why I used it before the link.  But I used the English colon at the end of the example descriptions per @mebeim's [comment](https://github.com/tldr-pages/tldr/pull/3442#pullrequestreview-303829335) a while back.

Reply:
I see. It's OK to me as this is due to tech reason. Looks like that was the reply to me while I totally forgot it 😂.